catalog/app/containers/Bucket/errors.tsx

```diff
@@ -167,7 +167,7 @@ const defaultHandlers: ErrorHandler[] = [
           <br />
           <StyledLink
             target="_blank"
-            href={`${docs}/walkthrough/working-with-the-catalog#access-control`}
+            href={`${docs}/quilt-platform-administrator/admin#users-and-roles`}
```

Comment:
this anchor was broken in https://github.com/quiltdata/quilt/commit/2c1aec8a5f5db4665db54fabeae5d61f9630f309#diff-c83d883e5d164b4035b039bb964282da0e39d8bafa72e2b193cd6760d8b976b1

I'm not sure if it should refer to the old place: https://docs.quiltdata.com/quilt-platform-catalog-user/working-with-the-catalog#admin-ui

Reply:
i think it's ok like this